en/_posts/2019-11-30-JS-testing-workshop.markdown

```diff
@@ -0,0 +1,53 @@
+---
+tags: intermediate js_testing_workshop
+categories: en events
+i18n_key: js_testing_workshop_2019_11
+---
+
+![Photo of JavaScript code on a laptop with cake on the side and an apple and lamp in the background.](/assets/images/JStesting-Nov19.jpg)
+
+## Let’s dive deeper into Coding ♥️
+
+You know how to code in JavaScript and want to learn testing with JS or improve your testing?
+
+#### When: **30.11.2019, 09:30 - 16:00**
+#### Where: **bitcrowd, Oranienstr 6, 10997 Berlin**
+
+#### Apply:
+The workshop is geared towards women* with coding experience in JavaScript. [**Apply Here**](https://forms.gle/dwyLkQXRsnVHGxnx6)
+
+Are a JavaScript coach? [**Apply here to coach at this workshop**](https://forms.gle/grVdR7y7hhZ9GCmu8).
+
+#### Registration Schedule
+- Registration starts: **13 November**
```

Comment:
I don't know how it is usually handled. I noticed that the way you've written the date isn't perfectly clear. 
Maybe we could use something like “November 13th”? or “13. November”?